remote_config/integration_test/src/integration_test.cc

```diff
@@ -266,6 +266,7 @@ TEST_F(FirebaseRemoteConfigTest, TestSetDefault) {
 TEST_F(FirebaseRemoteConfigTest, TestAddOnConfigUpdateListener) {
   ASSERT_NE(rc_, nullptr);
 
+  FLAKY_TEST_SECTION_BEGIN();
```

Comment:
As a reminder, does this flake on all platforms? If not you can surround these macros with #ifdefs ...

Reply:
The initial bug report is for the test being flaky on Android. I'll add an IF ANDROID check here and if it ends up flaking on other platforms we can change it.